<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>561</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>787</width>
    <height>561</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>787</width>
    <height>561</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ignition</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/SR_icon.png</normaloff>:/SR_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="repo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>731</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Repository</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="repoURL">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>571</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>http://192.168.15.2/content/trial.tar.gz</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showAll">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show all distributions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="repoSync">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>30</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Sync</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>413</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Network Status</string>
    </property>
    <widget class="QLabel" name="ethIp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="wlanIp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="install">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>420</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Install</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="customArea">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>191</width>
      <height>81</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>189</width>
       <height>79</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
     <widget class="QCheckBox" name="custom_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="custom_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="custom_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="custom_0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>CheckBox</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="osList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>731</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="debugButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Debug</string>
    </property>
   </widget>
   <widget class="QPushButton" name="configWifi">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>460</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Setup WiFi</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionEnable_Debug">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Debug</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>osList</tabstop>
  <tabstop>repoURL</tabstop>
  <tabstop>install</tabstop>
  <tabstop>customArea</tabstop>
  <tabstop>custom_1</tabstop>
  <tabstop>custom_2</tabstop>
  <tabstop>custom_3</tabstop>
  <tabstop>custom_0</tabstop>
  <tabstop>debugButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>osList</sender>
   <signal>itemActivated(QListWidgetItem*)</signal>
   <receiver>install</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repoURL</sender>
   <signal>returnPressed()</signal>
   <receiver>repoSync</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>700</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>syncRepoClicked()</signal>
  <signal>repoSyncURL()</signal>
 </slots>
</ui>
